src/routes/search/components/ThemedSearchBar.js

```diff
@@ -12,7 +12,7 @@ type PropsType = {|
 
 class ThemedSearchBar extends React.Component<PropsType> {
   render () {
-    const { theme, onChangeText, value } = this.props
+    const { theme, onChangeText, value, autofocus } = this.props
```

Comment:
Please add `autofocus` to the `PropsType` above. You probably don't have flow configured yet, but you can do this IntelliJ by setting:
Settings > Languages & Frameworks > JavaScript and
    * choose *Flow* as Language version
    * set *Flow package or executable* to *<project_dir>/node_modules/flow-bin*

Reply:
After doing this, flow type errors should be shown in the editor.